packages/typespec-client-generator-core/src/package.ts

```diff
@@ -64,7 +64,6 @@ import {
   getCrossLanguageDefinitionId,
   getCrossLanguagePackageId,
   getDefaultApiVersion,
-  getEffectivePayloadType,
```

Comment:
Deco getEffectivePayloadType?

Reply:
I think java won't use this. So, agree on deprecation.

Reply:
it is still used in response.